src/components/molecules/NavMenu/index.tsx

```diff
@@ -18,7 +19,7 @@ const NavMenu: React.FC<NavMenuInterface> = (
     return (
         <StyleMenuLinkContainer>
             {props.links.map((navLink) => (
```

Comment:
where is  `links` in the prop validation?

Reply:
Sorry, I don't understand what you're saying. I'm getting the list of pages from "en.json" (under "menu") using "useHomepageTL.js". I pass that array of objects down to the nav components and then map over them.

Am I supposed to be doing it differently?